client/ayon_houdini/plugins/publish/collect_local_render_instances.py

```diff
@@ -134,17 +134,23 @@ def process(self, instance):
                                                context,
                                                colorspace=colorspace)
 
+            families = ["render.local.hou"]
+            if instance.data.get('review'):
+                families.append("review")
+
             aov_instance.data.update({
                 # 'label': label,
                 "task": instance.data["task"],
                 "folderPath": instance.data["folderPath"],
                 "frameStart": instance.data["frameStartHandle"],
                 "frameEnd": instance.data["frameEndHandle"],
+                "frameStartHandle": instance.data["frameStartHandle"],
+                "frameEndHandle": instance.data["frameEndHandle"],
                 "productType": product_type,
                 "family": product_type,
                 "productName": product_name,
                 "productGroup": product_group,
-                "families": ["render.local.hou", "review"],
+                "families": families,
```

Comment:
```suggestion
                "families": ["render.local.hou", "review"],
```

Reply:
We do want the behavior to work - the question is whether your fix is the 'correct' approach to resolving it or that it's maybe hiding another issue in another area.

Reply:
ohh sorry now i see

but i feel like transfering families cant be bad or worse then hardcoding them

and i understand that somewhere we have tags but as someone that thee mount in setting everything up i met tags so rare (maybt i'm just dumb haha) but famalies is everywhere so kitsu just ignoring tags and using unly famalies as i remember when i was digging in collect kitsu family

Reply:
i want to add product name filtering! but i think that here changes i make nessasry to be here because why do we need families that was not in original instance?